standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -152,7 +151,7 @@ object StandardizationJob {
   private def applyCobolOptions(dfReader: DataFrameReader,
                                 cmd: CmdConfig,
                                 dataset: Dataset): DataFrameReader = {
-    val isXcom = cmd.cobolOptions.map(_.isXcom).getOrElse(false)
+    val isXcom = cmd.cobolOptions.exists(_.isXcom)
```

Comment:
Yes, IDEA suggested this change. But I liked the original better since intent is more expressive there.

Reply:
Yes, it's from IDEA suggestion. But I admit, I  find the suggestion good (most of them I actually do 😉 ) - to me it's seems clear, clearer then the original.

PS
I touched the file during some debugging. Thought it good to keep. Can _revert_ in general preference is there. 😃

Reply:
If you guys like it, I'm fine. Just think what you need to do to make the default value `true`.